contracts/src/bridge/Outbox.sol

```diff
@@ -44,19 +57,19 @@ contract Outbox is DelegateCallAware, IOutbox {
     /// @notice When l2ToL1Sender returns a nonzero address, the message was originated by an L2 account
     /// When the return value is zero, that means this is a system message
     /// @dev the l2ToL1Sender behaves as the tx.origin, the msg.sender should be validated to protect against reentrancies
-    function l2ToL1Sender() external view override returns (address) {
+    function l2ToL1Sender() external view override outboxIsRunning returns (address) {
         return context.sender;
     }
 
-    function l2ToL1Block() external view override returns (uint256) {
+    function l2ToL1Block() external view override outboxIsRunning returns (uint256) {
         return uint256(context.l2Block);
     }
```

Comment:
if we change the return values to this pattern, then it wont be a breaking change
```suggestion
    function l2ToL1Block() external view override outboxIsRunning returns (uint256 l2Block) {
        l2Block = context.l2Block;
        if(l2Block == EMPTY_CONTEXT.l2Block) return 0;
        return uint256(l2Block);
    }
```